### DISCUSSION TITLE
A little help about networking setup for SO under Virtual Box on a Windows Host

### DISCUSSION THREAD

**Discussion by mrdkx0101** [Security-Onion-Solutions/securityonion#7068] [Unsupported Versions] [answered]
Good afternoon and thanks for this great product and to whoever is going to spend some time reading and answering this.
- I have a Windows 10 Pro machine where i have installed VirtualBox. 
- Into VirtualBox, i run Ubuntu Server 20.04. Into Ubuntu Server i have installed SO. 
- I have 2 physical NICs, one for management, one for monitoring/sniffing. I have assigned a static IP to both NICs under Windows.
1. On Ubuntu Server i have setup 1st NIC (management) with same IP address as the one used into Windows. For 2nd NIC (monitoring) i have removed the IP/Gateway/Subnet/DNS entries on netplan (NetworkManager file has no entries).
2. Into VirtualBox settings i have setup 1st NIC as bridged adapter for management and 2nd NIC as bridged adapter for monitoring/sniffing. 
3. On VirtualBox Network Settings i have set promiscuous mode to "allow all" for 2nd NIC.

My question: 
the 2nd NIC used for monitoring/sniffing, how should be setup under Windows? Should i leave it assigned with a static IP address?
Because i didn't find an asnwer for myself i was also thinking to try and enable promiscuous mode on 2nd NIC under Windows (not sure if it is the right thing to do) but i cannot find a proper procedure to follow.

Am i missing something?

Thanks a lot, C.

**Comment by dougburks** [comment, marked as answer]
That really depends on what you're trying to do and whether you're trying to sniff traffic from the Windows machine itself or from a tap or span port. If you're unsure and just want to get started with Security Onion, we recommend just doing an Import installation and then importing pcap and/or EVTX files:
https://docs.securityonion.net/en/2.3/first-time-users.html
